src/components/Empty/index.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Empty } from '.';
+
+export default {
+  title: 'Elements/Empty',
+  component: Empty,
+  argTypes: {
+    children: { control: 'text' },
+  },
+} as ComponentMeta<typeof Empty>;
+
+export const Default: ComponentStory<typeof Empty> = (args: any) => {
+  return (
+    <Empty
+      description='No data'
+      image='https://user-images.githubusercontent.com/507615/54591670-ac0a0180-4a65-11e9-846c-e55ffce0fe7b.png'
```

Comment:
can you make this image the default image we use, even when we don't specify an image at all? also upload it to this component folder and get the image locally. This way in the event that this link is no longer valid, it does not effect the consumers of this component

Reply:
so remove the image prop and use that link as img src?